packages/react-i18n/src/i18n.ts

```diff
@@ -1,4 +1,5 @@
 import {languageFromLocale, regionFromLocale} from '@shopify/i18n';
+import memoizeFn from 'lodash/memoize';
```

Comment:
@michelleyschen can I use that new memoize function here? Not sure if it's ready yet

Reply:
I am going to published it toady :)
however, you may not want to use it since it doesn't run in server.